docs/advanced/kiali.adoc

```diff
@@ -0,0 +1,187 @@
+kiali-7966d98797-77gqz                      0/1       Running     0          1d
+prometheus-586d95b8d9-r6d6x                 0/1       Running     0          1d
+----
+
+If the "STATUS" column for Kiali is "Running", you can check which URL it is
+using by using this command:
+
+[source, bash]
+----
+oc project istio-system
+oc get routes
+----
+
+And you'll see a result that's something like this:
+
+[source, bash]
+----
+NAME                   HOST/PORT                                                PATH      SERVICES               PORT              TERMINATION   WILDCARD
+grafana                grafana-istio-system.127.0.0.1.nip.io                          grafana                http                            None
+istio-ingress          istio-ingress-istio-system.127.0.0.1.nip.io                    istio-ingress          http                            None
+istio-ingressgateway   istio-ingressgateway-istio-system.127.0.0.1.nip.io             istio-ingressgateway   http                            None
+jaeger-query           jaeger-query-istio-system.127.0.0.1.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.127.0.0.1.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.127.0.0.1.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.127.0.0.1.nip.io                          tracing                tracing           edge          None
+----
+
+So now we have the URL for Kiali (usually,
+"kiali-istio-system.127.0.0.1.nip.io"), so let's access it:
+
+image:docs/images/kiali-login.png[Login Page]
+
+The default credentials are "admin/admin", but it's recommended to change them
+before using it in production.
+
+== Generating Sample Data
+
+To show the capabilities of Kiali, you'll need an Istio-enabled application to
+be running. For this, we can use the `customer-tutorial` application we created
+earlier.
+
+To generate data for it, we can `curl` it with this command:
+
+[source, bash]
+----
+curl customer-tutorial.$(minishift ip).nip.io
+----
+
+== Service Graph
+
+After you login, you should see the Service Graph page:
+
+image:docs/images/kiali-service-graph.png[Service Graph Page]
+
+It shows a graph with all the microservices, connected by the requests going
+through then. On this page, you can see how the services interact with each
+other.
+
+== Service Listing
+
+Click the Services link in the left navigation. On the Services page you can
+view a listing of all the services that are running in the cluster, and
+additional information about them, such as health status.
+
+image:docs/images/kiali-service-list.png[Service Listing Page]
+
+Click on the "Preferences" service to view the Service Details page, where you
```

Comment:
"Preferences" is not visible. Instruct people to look for the "preference" service in the search box.